src/ui/component/walletBackup/view.jsx

```diff
@@ -55,6 +146,25 @@ class WalletBackup extends React.PureComponent<Props> {
                 <Button button="link" href="https://lbry.com/faq/how-to-backup-wallet" label={__('see this article')} />
                 .
               </p>
+              <p className={'card__message card__message--error' + (this.state.errorMessage ? '' : ' hidden')}>
+                {this.state.errorMessage}
+              </p>
+              <p className={'card__message card__message--success' + (this.state.successMessage ? '' : ' hidden')}>
+                {this.state.successMessage}
+              </p>
+              <div className="card__actions">
+                <Button
+                  button="primary"
+                  label={__('Copy Folder Path')}
+                  onClick={() => this.copyWalletDirToClipboard(lbryumWalletDir)}
```

Comment:
Instead of showing the success message for copy, lets use `doToast({ message: "..." })`, that's what we use for other places that have a copy button. It also won't remove the success message after creating a backup.

Reply:
Hi @seanyesmunt , when looking through the source for other uses of doToast, I noticed some components (like the wallet address) use a CopyableText component that provides the copy button right next to the text field. I have replaced the copy button I added with the use of this CopyableText component.

Also, I thought I should mention, I haven't been able to test on the master branch due to an `npm install` error referencing an unknown commit in lbry-redux: b3bf3f6d53410ff1c5415b51ca425341e364959f, so I've been testing on 0.33.2 for now.

Thanks!

Reply:
@jcamp0x2a sorry about that. I just pushed a commit that fixes that. Should be good to go now.